src/main/java/xyz/niflheim/stockfish/engine/UCIEngine.java

```diff
@@ -22,15 +24,22 @@
 import java.io.*;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
+
+import static xyz.niflheim.stockfish.engine.util.Util.*;
```

Comment:
I think it's better not to use static imports

Reply:
Implemented as you requested